Dockerfile

```diff
@@ -127,7 +134,7 @@ USER tenzir:tenzir
 
 EXPOSE 5158/tcp
 WORKDIR /var/lib/tenzir
-VOLUME ["/var/lib/tenzir"]
+VOLUME ["/var/cache/tenzir", "/var/lib/tenzir"]
```

Comment:
This also needs to be added to the Docker Compose file, both in this repository and for the platform-generated one.